model/dao/ProjectDAO/PostgreSQLProjectDAO.php

```diff
@@ -568,6 +568,18 @@ public function getAllCustom($active = False, $orderField = 'id') {
         return $this->customExecute($sql);
     }
 
+    public function getByDescription(string $desc): ?int
+    {
+        $sql = "SELECT * FROM project WHERE description='" . $desc . "'";
+
+        $res = pg_query($this->connect, $sql);
```

Comment:
With the disclaimer that I worked with and developed a bit of an open-source crush on @anarute (sorry, Ana; I have to add this comment because I looked at this PR and can't in good conscience ignore it; I'm not following your day-to-day GitHub contributions, and this observation was a one-off)

It might be worth looking at [`pg_query_params`](https://www.php.net/manual/en/function.pg-query-params.php) to reduce the risk of SQL injection here; it should handle escaping of parameters to the query.